.github/workflows/ci.yml

```
@@ -38,6 +38,23 @@ jobs:
           name: install-build-artifact-node-${{ matrix.node-version }}
           path: install-build-node-${{ matrix.node-version }}.tar
           retention-days: 1
+  install-and-build-on-windows:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16, 20]
+    name: Upload install and build artifact on Windows (Node ${{ matrix.node-version }})
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3.7.0
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+      - name: Install dependencies and build sequelize
```

```suggestion
      - name: Install dependencies
```
Nitpicky, but can you change this for the Ubuntu install and build job as well?